docs/rules/unicode-bom.md

```diff
@@ -0,0 +1,65 @@
+# Require or disallow the Unicode Byte Order Mark (BOM) (unicode-bom)
```

Comment:
Okay, looks like we aren't going to be changing the header at this point, so we're good to go.